geode-core/src/main/java/org/apache/geode/distributed/internal/InternalLocator.java

```diff
@@ -1334,6 +1335,12 @@ public void onConnect(InternalDistributedSystem sys) {
     try {
       this.stats.hookupStats(sys,
           SocketCreator.getLocalHost().getCanonicalHostName() + '-' + this.server.getBindAddress());
+      ClientProtocolMessageHandler messageHandler = this.server.getMessageHandler();
+      if (messageHandler != null) {
+        // TODO GEODE-3546 - this should create locator-specific stats but is creating client/server
```

Comment:
Can we please keep the TODO out? I am concerned that this will just add to the noise that we already have from the other 2,400+ TODOs in the code base. We have a story and hopefully it gets prioritized and we can address this. There is really no place in our process where we prioritize TODOs.